### PR TITLE
fix(localdebug): hot reload when json file changed

### PR DESCRIPTION
**Pull request by XiaofuHuang**
Detail: [Bug 14264065](https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/14264065): Update adaptive card doesn't get refreshed in hot reload

### CODE REVIEW

**Comment by XiaofuHuang** [comment]
E2E TEST: https://github.com/OfficeDev/TeamsFx-UI-Test/blob/328a68696f5a62d43251318c5fc46490bf44aad3/src/ui-test/localdebug/preview%40localdebug-notification-restify.test.ts#L12

**Review by yukun-dong — DISMISSED** [review]

**Review by kuojianlu — APPROVED** [review]

File: templates/bot/ts/command-and-response/package.json [hunk 1]
```diff
@@ -7,10 +7,10 @@
     "main": "./lib/index.js",
     "scripts": {
         "dev:teamsfx": "env-cmd --silent -f .env.teamsfx.local npm run dev",
-        "dev": "nodemon -e ts --exec node --inspect=9239 --signal SIGINT -r ts-node/register ./src/index.ts",
+        "dev": "nodemon --exec node --inspect=9239 --signal SIGINT -r ts-node/register ./src/index.ts",
```
Comment (swatDong) [inline templates/bot/ts/command-and-response/package.json:10]: I remember we exclude json file due to ignoring `.notification.localstore.json`, will this monitor the json file as well?

Comment (swatDong) [reply templates/bot/ts/command-and-response/package.json:10]: How about using `nodemon.json` to control details? To ensure that only some required `json` files are in monitor scope.

Comment (XiaofuHuang) [reply templates/bot/ts/command-and-response/package.json]: How about make `dev` script same to the `watch` script? Only the files under `src/` are watched.
Maybe user code contain some other kinds of files under `src` folder. (e.g. yml / html / ...... )

**Review by swatDong — APPROVED** [review]

**Pull request merged**